### ISSUE TITLE
ENH: Add equivalent of `mne_organize_dicom`

### ISSUE THREAD

**Issue by larsoner** [mne-tools/mne-python#6816]
There are refs in the C manual to `mne_organize_dicom` for FLASH BEM use. Should we make a Python equivalent? It's the one thing that was missing while removing `mne_*` from the MNE-Python docs.

**Comment by larsoner** [comment]
I'm actually thinking I should close this as `wontfix`. I have successfully used `dcm2niix` with Philips DICOM files so presumably the same thing will work for Siemens, and it should be fairly straightforward for people to do given our existing docs of what needs to be where for `mne flash_bem`. After #6809 hopefully it's clear enough for people and we can just iterate as they find problems.
